### ISSUE TITLE
clippingPolygons  Performance issues

### ISSUE THREAD

**Issue by qq1066536** [CesiumGS/cesium#12177]
### What happened?

![image](https://github.com/user-attachments/assets/351c1ec9-9325-46ef-95d9-7d1ed7b30913)
open clip ,Performance  cpu up to100%

### Sandcastle example

https://sandcastle.cesium.com/#c=nVbbbhs3EP0VQujDGnColZW48RVNbddx4DSGLeQhVYFQy9GKMZcjkFy5SuB/75Dc1c1ym+RFK5JzPXNmyAKN82ym4AEsO2EGHtgZOFVX/GPcy4adIq7P0HihDNhhZ5d9GxrGvKpA084hGwvtYDfsCaMq4RWa1U1XgIH3KOFGFfdgV49GwsG1mIN9etTtssEEWKlxBEwiOGbQU4AgmUcW9pSbatKVrbhTpgDmSelmgh4tCK2cVwXrn7OB0mSBBHQtSQaspWSCXrS/9Pu4czQ0CQ4e4+YSRnV5N8GHP6yowN2AvYMCjWQn3tZA0kNDvi+MGGlgFowEq0wZw3D38w1jtPPGV+imE7C0IqsE+sJOEYtR1JbM+AHBS4dNNd7VWglzLjzwscXqyuHr/byXhQyGnb18L3+R917kB4O9/mGeH/YPeN57vbffO6CfPD/Y7/Vf7n8adoZmJb1CY3HP172trNrM3kj5HJ5Do8GzErHUEDfAk5a380SQtQMyLh6E8m1CBZnzcBlF1s33zxuVLATL2BqAU6sq5dUMHBdSZmsugvgjK4QvJiyjCqPdSYEEXFED11hmny/CAdMoZCjTc0zxySQP6r98i8YeP0cHDSzXZIAJdgn47u7Dn2xM8uxB+QmbolOhAxyTMFam5cLU4hcoPBsjekrC+ATeYrkKXOKBBYe1LWATuSs0t81R5MIbR4FeyWzvVb/f6/cSaMmEFF7cbTUS4nYYCNUI8IBI1vpsWpzsaFFNB3hpsTbyMFI1tlvsk5XqLD2lwizXreAiUwplecqJbQQXac2ErulDkMksi7vzHXZymsJooo6IgmyO+RT1vESzE2RS2gsn7RnH2ochRU5jizfBUAE/IVZhkoRYiA5FHFu7NMEkC3QNNZsgNUVIcIloQTPAig/jcaL0yrx8C5FRN4rodytMCdla6O+Fn3CPtyQkjMsOXvF8Z/e/JF78uiLyOs95vkyzAf0rpTDAbJH07lp4a6Ihld9r7wmRsBOmseYFVhXly0cwph64+AeKmuYLle9iRghfU0tQy9kmjwxCOVpeMAa8EDRv9XKApf0fiI1o1Ob0o31bAn4h/vLtzRlmliDuquk0CFsoqbbxspGM/oTaBkO02tKWqdDLNj7ZwqqJolRsMZnzEvzHwNxshy9UjhajvIngJqm5dcacrZ+eodYUChnIYvaNL3fI/kpgPa+aLaqyiOFw+bfh0GMi09/tLbc2O/mWUDe3Vm+EZj4yHAcIydUKlpJiLE2ab6Na6VCwLdfDxtHmhEqfxWWQrj007bRLOcWZt/fySWdsvSw2HP4M7VJq/3M73FFPFcJ5HXtpgCXBnHqPXlPhJbGErIGKnlRxsrJxbSIDWMyvmAC9iiTd2lsRa98PjdhRU9fvCWBJ9y2uG3uN0yX5f8pdYCrzwlKfLKbs9+a7SUC6LMIrS/5wEFdmBtZBNBicp9fesyk/8asa/Sd+O7udY+fnGk7bBvxNVVO0ntVWZ5x3PVT0SPXguqOaFKnPnGuH33F3VfVYqhlT8mTLgzvcws7RybjW+k59hWHn9LhL8k9UG55+oHDpaRzEJr3T67TJOT/u0nK7pkfUI2E3LP8L

**Comment by ggetz** [comment]
Thanks for the report @qq1066536!

**Comment by jcuda** [comment]
@ggetz Maybe this could be closed as a duplicate of https://github.com/CesiumGS/cesium/issues/12258

**Comment by ggetz** [comment]
Yes, thanks @jcuda! Closing this as a duplicate in favor of https://github.com/CesiumGS/cesium/issues/12258.

**Comment by javagl** [comment]
Wrong account. That was me. Too many accounts...
